calico.yaml

```diff
@@ -0,0 +1,446 @@
+package:
+  name: calico
+  version: 3.26.0
+  epoch: 2
```

Comment:
this is +1 of the old `calicoctl` package to avoid any conflicts with `calicoctl`